Makefile

```diff
@@ -78,6 +78,12 @@ prepare-integration-tests:
 		@cd IntegrationTests/ && pod install
 		@echo "OK 👌"
 
+open-sr-snapshot-tests:
+		@echo "⚙️  Opening SRSnapshotTests with DD_TEST_UTILITIES_ENABLED ..."
+		@pgrep -q Xcode && killall Xcode && echo "- Xcode killed" || echo "- Xcode not running"
+		@sleep 0.5 && echo "- launching" # Sleep, otherwise, if Xcode was running it often fails with "procNotFound: no eligible process with specified descriptor"
+		@open --env DD_TEST_UTILITIES_ENABLED ./DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests.xcworkspace
+
```

Comment:
Well, I'm still not convinced by how practical this is, or will be for other UI Tests.

Killing all my instances of Xcode for running an inner project is not a good experience IMO. I found CP setup more straightforward, we could run all `pod install` in the `make dependencies` and we would be set!

Reply:
I see the point 👍, although I'm more on the side of this simple setup rather than the impact of adding CP to a project. The effect of "killing Xcodes" is mitigated by the fact that we can't have multiple projects open anyway if they link the same local package as the workspace dependency.

This is rather a topic for a wider debate on how we want to manage dependencies in the repo. I can take a lead on that and gather all options in an RFC. With V2 in place, we see better what are the needs vs limitations 🙂.